Sources/XiEditor/EditViewController.swift

```diff
@@ -946,6 +946,12 @@ extension EditViewController: NSWindowDelegate {
     func windowDidResignKey(_ notification: Notification) {
         editView.isFrontmostView = false
     }
+    
+    @objc func windowShouldClose(_ sender: NSWindow) {
```

Comment:
I'm getting some compiler warnings in this function?

Reply:
An apple engineer on the dev forums said it's a Xcode bug.